packages/app-builder-lib/package.json

```diff
@@ -93,10 +95,9 @@
     "@types/is-ci": "^2.0.0",
     "@types/js-yaml": "^3.12.6",
     "@types/normalize-package-data": "^2.4.0",
-    "@types/semver": "^7.3.4",
-    "dmg-builder": "workspace:*",
-    "electron-builder-squirrel-windows": "workspace:*"
+    "@types/semver": "^7.3.4"
   },
-  "//": "electron-builder-squirrel-windows and dmg-builder added as dev dep for tests (as otherwise `require` doesn't work using Yarn 2)",
-  "typings": "./out/index.d.ts"
+  "//": "electron-builder-squirrel-windows and dmg-builder added as deps (as otherwise `require` doesn't work using Yarn 2)",
```

Comment:
This comment shouldn't exist as this is true for node_modules as well (Yarn 2 !== Yarn PnP btw), hoisting just happens to make it work if you're lucky.
https://yarnpkg.com/advanced/rulebook#packages-should-only-ever-require-what-they-formally-list-in-their-dependencies